README.md

```diff
@@ -0,0 +1,9 @@
+# apps
+Proof of concept applications that use the Force Move Game framework.
+
+### Setting up development environment and running the app
+* Build `wallet-comm` package via `yarn build` in the package directory.
+* Install lerna via `npm i -g lerna`
```

Comment:
You should be able to just run `yarn` from the root directory (since the root `package.json` has an entry for `lerna`)